packages/demo/src/components/examples/LimitExamples.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import {Limit} from 'react-vapor';
+
+export class LimitExamples extends React.Component {
+    render() {
+        const title: string = 'Limit example';
+        const usageNumber: number = 42;
+        const usageNumberNearLimit: number = 80;
+        const usageNumberEqualToLimit: number = Limit.defaultProps.limitNumber;
+        const customLimitNumber: number = 130;
+        const customLimitTitle: string = 'Throttling limit';
+        const isLimitCanBeChanged: boolean = true;
+        const isLimitIsTheGoalToReach: boolean = true;
+        const isHistoryIsIncluded: boolean = true;
+
+        return (
+            <div className="coveo-form">
+                <div className="form-group">
+                    <label className="form-control-label"> A limit with an usage value</label>
```

Comment:
`>A limit...` (space)

Reply:
To make sure I understand your comment, you suggest that my line would look like this:
` <label className="form-control-label"> A limit with an usage value </label>`
or like this:
`<label className="form-control-label"> 
        A limit with an usage value
</label>`
?

Reply:
Sorry, I should have been clearer - you should remove the unnecessary space between the closing bracket and the letter A